src/app/components/Item/ItemHoldings.jsx

```diff
@@ -138,7 +138,9 @@ class ItemHoldings extends React.Component {
 
     return (
       <div id="item-holdings" className="item-holdings">
-        <h2>{this.props.title}</h2>
+        <dt className="list-multi-control">
```

Comment:
This `<dt>` is hanging out by himself. `{itemTable}` is returning a `<dl>` so don't think it's necessary? Need to check on the correct DOM structure for definition lists.